packages/client/hmi-client/src/types/Types.ts

```diff
@@ -710,6 +711,14 @@ export interface TimeSpan {
     end: number;
 }
 
+export interface TaskResponse {
+    id: string;
+    script: string;
+    status: TaskStatus;
+    output: any;
+    additionalProperties: any;
+}
+
```

Comment:
If these are client-only types they need to be put into their own files.  This is generated and the build will fail if it doesn't have a matching java type.

Reply:
Those were generated